release-notes/2.1/2.1.1-commit.md

```diff
@@ -22,7 +22,7 @@ This is a somewhat curated list of commits across the entire .NET Core 2.1.1 dev
 ## CoreFX
 
 * [`[3700c5b793]`](https://github.com/dotnet/corefx/commit/3700c5b793) Update to a xUnit Performance Api that has a bigger Etw buffer size. … (#30328)
-* [`[6b38470265]`](https://github.com/dotnet/corefx/commit/6b38470265) Use _SC_NPROCESSORS_CONF instead of _SC_NPROCESSORS_ONLN in Unix_ProcessorCountTest on ARM/ARM64 (#30132)
+* [`[6b38470265]`](https://github.com/dotnet/corefx/commit/6b38470265) Use _SC_NPROCESSORS_CONF instead of_SC_NPROCESSORS_ONLN in Unix_ProcessorCountTest on ARM/ARM64 (#30132)
```

Comment:
```suggestion
* [`[6b38470265]`](https://github.com/dotnet/corefx/commit/6b38470265) Use \_SC_NPROCESSORS_CONF instead of \_SC_NPROCESSORS_ONLN in Unix_ProcessorCountTest on ARM/ARM64 (#30132)
```